docs/checkit.js

```diff
@@ -783,6 +783,89 @@ CheckIt.prototype.findCurrentDay = function() {
 };
 
 
+// Should these be CheckIt functions? or Calendar. TODO
+// How to access the calendar object when I need it for these functions?
+// current active calendar...??
+
+CheckIt.prototype.getNumberOfChecked = function() {
+    // Returns the number of checked days in a calendar
+
+    var checkit = this;
+
+    return checkit.store.getActive().then(function(activeCalendarId) {
+        console.log("found this activeCalendarId: " + activeCalendarId);
+        return checkit.store.loadById(activeCalendarId).catch(function(err) {
+            console.log("Couldn't find active calendar specifically in getNumberOfChecked");
+            console.log("There is no current active calendar " + err);
+            checkit.store.removeActive();
+            return err;
```

Comment:
Returning a value here is going to "fix" the error and cause `.then()` to be called with `err` as the argument, isn't it?